daffodil-test/src/test/resources/org/apache/daffodil/section14/sequence_groups/SequenceGroup.tdml

```diff
@@ -1164,7 +1164,7 @@
       </xs:complexType>
     </xs:element>
     <xs:group name="g0">
-      <xs:sequence dfdl:separatorPosition="infix">
+      <xs:sequence>
```

Comment:
They can have properties and those are combined with those on the group reference to form the combined set of properties for that sequence.

Reply:
Then why did `dfdl:separatorPosition` need to be removed from these group sequences? I assumed it was removed because it was creating a unused property warning, but it seems like the property should be used?

Reply:
It was indeed creating an unused property warning, but since the default separatorPosition is infix. I deemed them unnecessary and safe to remove.

Reply:
Seems like that shouldn't create a warning. Even though it's the same as the default the property should still be used. It just means the default format won't be used. Maybe there's an additional code change needed?

Reply:
Ok, I'll add them back in and investigate what's going on